src/_incydr_sdk/actors/client.py

```diff
@@ -208,73 +193,69 @@ def update_actor(
 
         **Parameters**:
 
-        * **actor_id**: `str` (required) - Unique ID for an actor.
+        Either:
+
+        * **actor**: `str` (required) - Unique ID for an actor.
         * **notes**: `str` - Additional notes for the risk profile. Pass an empty string to clear the field.
         * **start_date**: `datetime` - The starting date for the user. Accepts a datetime object or a string in the format yyyy-MM-dd (UTC) format. Pass an empty string to clear the field.
         * **end_date**: `datetime` - The departure date for the user.  Accepts a datetime object or a string in the format yyyy-MM-dd (UTC) format.  Pass an empty string to clear the field.
 
-        **Returns**: An [`Actor`][actor-modhel] object representing the actor.
+        Or:
+
+        * **actor**: An [`Actor`][actor-model] object representing the actor.
+
+        **Returns**: An [`Actor`][actor-model] object representing the actor.
         """
-        ...
-
-    def update_actor(self, *args, **kwargs):
-        if len(args) == 1 and isinstance(args[0], Actor):
-            return _update_actor_from_actor(self, args[0])
-        elif set(kwargs.keys()).issubset(
-            {"actor_id", "notes", "start_date", "end_date"}
-        ):
-            return _update_actor_from_args(self, *args, **kwargs)
+        if isinstance(actor, Actor):
+            return self._update_actor_from_actor(actor)
```

Comment:
IMO totally cool to just ignore the kwargs in the case of the `Actor` object, but we may want to at least print something out (or maybe throw an exception?) if they pass any kwargs with it just to be like "updating actor using Actor model, ignoring all additional args..." or something like that to avoid confusion